Quantum-Control-Applications/README.md

```diff
@@ -0,0 +1,40 @@
+# Quantum Control Applications
+
+This folder contains scripts and examples for different qubits types, all the way from basic to advanced protocols. 
+It also includes various examples and results from labs as listed below.
+
+Note that for them to work, you would need to download the latest version of the [py-qua-tools](https://github.com/qua-platform/py-qua-tools#installation).
+
+## Superconducting Qubits
```

Comment:
I would add here first a link to all of the normal files (with linked to all of the readme files).
And then start with all of the usecases, maybe under a title such as "advanced use-cases"?

Reply:
I indeed wanted to do that but I got lazy... I will do it now :)